test/nlp/show.jl

```diff
@@ -3,7 +3,8 @@
   io = IOBuffer()
   show(io, nlp)
   showed = String(take!(io))
-  expected = """  SimpleNLPModel
+  storage_type = typeof(nlp)
```

Comment:
I directly took the type of nlp, because it looks like nightly uses spaces, I though it would be simpler like this

Reply:
I think that's fine.